reference/api/tasks.md

```diff
@@ -357,10 +357,10 @@ A valid `uids`, `statuses`, `types`, `indexUids`, or date(`beforeXAt` or `afterX
 | **`statuses`**         | Cancel tasks based on `status`. Separate multiple `statuses` with a comma (`,`). Use `statuses=*` for all `statuses`                 |
 | **`types`**            | Cancel tasks based on `type`. Separate multiple `types` with a comma (`,`). Use `types=*` for all `types`                            |
 | **`indexUids`**        | Cancel tasks based on `indexUid`. Separate multiple `uids` with a comma (`,`). Use `indexUids=*` for all `indexUids`. Case-sensitive |
-| **`beforeEnqueuedAt`** | Cancel tasks **before** a specified `enqueuedAt` date                                                                                |
-| **`beforeStartedAt`**  | Cancel tasks **before** a specified `startedAt` date                                                                                 |
-| **`afterEnqueuedAt`**  | Cancel tasks **after** a specified `enqueuedAt` date                                                                                 |
-| **`afterStartedAt`**   | Cancel tasks **after** a specified `startedAt` date                                                                                  |
+| **`beforeEnqueuedAt`** | Cancel tasks **before** a specified `enqueuedAt` date. Use `beforeEnqueuedAt=*` to cancel all tasks                                  |
```

Comment:
Shouldn't we write ``` `*` (all tasks)```like we did for all the other parameters?

Or at least I think we should use the word `select` instead of `cancel`. Because you can select everything in one field and then constraint the selection to only the `documentAddition` for example.